mlrun/db/httpdb.py

```diff
@@ -1686,7 +1686,7 @@ def get_builder_status(
             last_log_timestamp = float(
                 resp.headers.get("x-mlrun-last-timestamp", "0.0")
             )
-            if func.kind in mlrun.runtimes.RuntimeKinds.nuclio_runtimes():
+            if func.kind in mlrun.runtimes.RuntimeKinds.pure_nuclio_deployed_runtimes():
                 mlrun.runtimes.nuclio.function.enrich_nuclio_function_from_headers(
                     func, resp.headers
                 )
```

Comment:
Why?

Reply:
It's an implementation detail because application runtime uses `get_builder_status` when building the image with mlrun and then uses `get_nuclio_deploy_status` when building with nuclio so I don't want `get_builder_status` to override the `status.container_image` because the headers are not expected in that flow for application runtime.
I will add a comment in my next PR because I don't want to block releasing next RC.

Reply:
@alonmr gotcha, thanks 👍  Comment is indeed required here, but agree that can be done in the next PR.